test/blackbox-tests/test-cases/github660/run.t

```diff
@@ -4,21 +4,26 @@ When there are explicit interfaces, modules must be rebuilt.
   Entering directory 'explicit-interfaces'
           main alias runtest
   hello
-  $ echo 'let x = 1' >> explicit-interfaces/lib_sub.ml
+  $ echo 'let _x = 1' >> explicit-interfaces/lib_sub.ml
   $ dune runtest --root explicit-interfaces
   Entering directory 'explicit-interfaces'
           main alias runtest
   hello
 
+CR-someday diml: this is weird that we need to do that:
```

Comment:
That seems weird, what happens if this is not added? If that's not fixable immediately, I think we should replace this CR- comment by a link to a bug report.

Reply:
I added a test in https://github.com/ocaml/dune/pull/923

Reply:
For this PR I just changed the test to create the modified files in run.t

Reply:
Yes, that seems better that way indeed!